### ISSUE TITLE
EXC_BAD_ACCESS in DDOSLogger on specific text

### ISSUE THREAD

**Issue by Emailrus** [CocoaLumberjack/CocoaLumberjack#885]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [x] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | ios
 Platform Version        | 10.3
 CocoaLumberjack Version | 3.2.0
 Integration Method      | cocoapods
 Xcode Version           | Xcode 8.3.3
 Repro rate              | all the time (100%)
 Demo project link       | Text below

### Issue Description and Steps

* Use DDOSLogger in setup
* Try `DDLogError(@"%@", @"100% size");`

It seems actual string that is causing crash is "% s". And it crashes in os_log_error method, so not really a CocoaLumberjack issue, even though good old DDASLLogger and DDTTYLogger are working fine here.

In short you can cause a crash with this: `os_log_error(OS_LOG_DEFAULT, [@"% s" UTF8String]);`

**Comment by sushichop** [comment]
@Emailrus 
Thanks for reporting.

Now at least, the following macro works fine with `CocoaLumberjack 3.4.2`  and Xcode 9.4.1 at least.
```objective-c
DDLogError(@"%@", @"100% size");
```

In case of using `os_log_error` directly, see the following example.
```objective-c
os_log_error(OS_LOG_DEFAULT, "%s", "100% size");
```

Thanks.

**Comment by Emailrus** [comment]
Yep it seems issue was resolved
